package.json

```diff
@@ -5,7 +5,8 @@
   "keywords"      : ["model", "view", "controller", "router", "server", "client", "browser"],
   "author"        : "Jeremy Ashkenas <jeremy@documentcloud.org>",
   "dependencies"  : {
-    "underscore"  : ">=1.6.0"
+    "underscore"  : ">=1.6.0",
+    "jquery"      : ">=2.0.0"
```

Comment:
yep it should be.

jquery 2.x only supports ie 9+ so this would be a pretty nasty breaking change to this lib for many people.